### PR TITLE
Added persist option to wan-replication-ref

### PR DESCRIPTION
**Pull request by ahmetmircik**
needed by this PR: https://github.com/hazelcast/hazelcast/pull/13281

### CODE REVIEW

**Comment by ahmetmircik** [comment]
run-lab-run

**Comment by ahmetmircik** [comment]
verify

File: hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/DynamicConfigTemplate.java [hunk 1]
```diff
@@ -380,37 +383,42 @@ void addReliableTopicConfig(String name, @Nullable List<ListenerConfigHolder> li
...
+     *                                          number of members required in the cluster for the cache to remain functional.
+     *                                          When {@code null}, quorum does not apply to this cache's operations.
+     * @param mergePolicy                       name of a class implementing {@link com.hazelcast.cache.CacheMergePolicy}
+     *                                          that handles merging of values for this cache while recovering from
+     *                                          network partitioning
+     * @param disablePerEntryInvalidationEvents when {@code true} disables invalidation events for per entry but
+     *                                          full-flush invalidation events are still enabled.
+     * @param partitionLostListenerConfigs      partition lost listener configurations
+     * @param expiryPolicyFactoryClassName      expiry policy factory class name. When configuring an expiry policy,
+     *                                          <<<<<<< Updated upstream
```
Comment (sancar) [inline hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/DynamicConfigTemplate.java]: Leftover from merge conflict

Comment (ahmetmircik) [reply hazelcast/src/main/java/com/hazelcast/client/impl/protocol/template/DynamicConfigTemplate.java]: fixed

**Comment by ahmetmircik** [comment]
no need this PR any more.

**Pull request closed without merging**